util/chplenv/chpl_qthreads.py

```diff
@@ -23,8 +23,8 @@ def get_compile_args():
 #  (linker_bundled_args, linker_system_args)
 @memoize
 def get_link_args():
+    (bundled, system) = third_party_utils.get_bundled_link_args(
```

Comment:
I don't think we should be ignoring qthreads deps. Let's open an issue with the qthreeads team to request a pkg-config file. In the meantime, I'm not sure what to do here.

Reply:
Would it be OK with you if we made this use `libtool_get_bundled_link_args` in the near term? The AUR package won't actually be able to use libtool, and it will run the fallback, but that's all I had here and I think it will work in typical configurations (so should be good enough for AUR for now).

Reply:
yeah

Reply:
I have created https://github.com/Qthreads/qthreads/issues/107 for qthreads.